contents.json

```diff
@@ -5454,6 +5454,11 @@
       "category": "utility",
       "description": "run swift code on slack.",
       "homepage": "https://github.com/noppefoxwolf/Swiftbot"
+    }, {
+      "title": "CollapsibleTableSectionViewController",
+      "category": "uitableview",
+      "description": "A swift library to support collapsible sections in a table view.",
```

Comment:
@jeantimex Could you remove "Swift" from description?

Reply:
Done! Thanks for the review!